tests/integration/test.all_docs.js

```diff
@@ -460,6 +460,129 @@ adapters.forEach(function (adapter) {
 
     });
 
+    it('test total_rows with a variety of criteria * 100', function (done) {
+      var db = new PouchDB(dbs.name);
```

Comment:
```suggestion
      const db = new PouchDB(dbs.name);
```

Reply:
This test is a copy of the test above, and formatting & variable scoping are deliberately unchanged.